src/base/types/StandardModel.jl

```diff
@@ -382,3 +382,86 @@ function Base.convert(::Type{StandardModel}, model::MetabolicModel)
         genes = modelgenes,
     )
 end
+
+#TODO generalize these to other model types
+
+"""
+    reaction_bounds(model::StandardModel, rid::String)
+
+Return lower and upper bounds for `rid` in `model`.
+"""
+function reaction_bounds(model::StandardModel, rid::String)
+    model.reactions[rid].lb, model.reactions[rid].ub
+end
```

Comment:
this way of accessing the stuff is super-specific to the OOP models, I'll think about ways to make it simpler